### PR TITLE
enable google oauth on ui

### PR DESCRIPTION
**Pull request by vijayrathore8492**
Signed-off-by: Vijay Kumar Rathore <professional.vijay8492@gmail.com>

## Change Summary

Enable google oauth.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/pages/signin.vue [hunk 1]
```diff
@@ -121,6 +121,19 @@ function resetError() {
...
+        <div>
+          <a-button
+            v-if="appInfo.googleAuthEnabled"
+            :href="`${appInfo.ncSiteUrl}/auth/google`"
+            outlined
+            large
+            elevation-10
+            block
+            color="blue"
+          >
```
Comment (bcakmakoglu) [inline packages/nc-gui/pages/signin.vue:133]: For styling reasons could you use this instead?

```
<a
  v-if="appInfo.googleAuthEnabled"
  :href="`${appInfo.ncSiteUrl}/auth/google`"
  class="scaling-btn bg-opacity-100 after:(!bg-white) !text-primary !no-underline"
>
  <span class="flex items-center gap-2">
    <LogosGoogleGmail />

    Sign In with Google
  </span>
</a>
```

**Review by bcakmakoglu — CHANGES_REQUESTED** [review]

File: packages/nc-gui/pages/signin.vue (hunk 1 above)
Comment (vijayrathore8492) [reply packages/nc-gui/pages/signin.vue:133]: Screenshot 2022-10-05 at 15 08 08
It looks like this

**Comment by bcakmakoglu** [comment]
Replaced by #3945

File: packages/nc-gui/pages/signin.vue (hunk 1 above)
Comment (bcakmakoglu) [reply packages/nc-gui/pages/signin.vue:133]: I opened a new PR as suggested :) 
Thank you for your contribution!

**Pull request closed without merging**